src/ansys/fluent/core/search.py

```diff
@@ -584,15 +537,19 @@ def _search_semantic(search_string: str, language: str, api_tree_data: dict):
                 ):
                     similar_keys.add(api_object_synset_name + "*")
     if similar_keys:
+        results = []
         for key in similar_keys:
-            _search_wildcard(key, api_tree_data)
+            result = _search_wildcard(key, api_tree_data)
```

Comment:
We are searching by api-names. So, the exact string-search should work here?

Can the result contain duplicate? Multiple api-name can belong to same api-path.

Reply:
exact string-search will work here but it will loose semantic search meaning as wildcard search shows all possibilities but this is not the case with exact-search therefore we have used wildcard search here.

 Can the result contain duplicate? - No. Now we are showing results that ends with the given search string.